sycl/test-e2e/Matrix/joint_matrix_bfloat16_16x16x16.cpp

```diff
@@ -17,9 +17,9 @@
 using namespace sycl;
 using namespace sycl::ext::oneapi::experimental::matrix;
 
-#define SG_SZ 16
-#define TM 16
-#define TN 16
-#define TK 16
+constexpr size_t SG_SZ = 16;
```

Comment:
Please, keep #define SG_SZ 16, as we will need those defines for SG=32 for conditional code under macros.